docs/osf-trivia-list/dasharo.md

```diff
@@ -235,6 +235,83 @@ Dasharo Tools Suite
+
+Dasharo repository contains release tags which are associated with Dasharo
+Certification Program providing certain quality criteria including test
+results. We always trying to minimize delta, but sometimes it can be up to 5k
+SLOC (or more I guess e.g. Talos II coreboot support).
+
+## What is Dasharo Certification Program?
+
+The Dasharo Certification Program (DCP) is a highly specialized certification
+program that benchmarks open-source firmware ecosystem deliverables. The
+program ensures that firmware is stable, secure, and dependable while aligning
+with the Dasharo values. DCP encourages developers to create their version of
+Dasharo or contribute to the Dasharo project or coreboot upstream, enabling
+them to leverage the power of open-source development to create custom firmware
+tailored to their specific needs based on years of Dasharo quality assurance
+results. The program's rigorous certification process entails comprehensive
+testing in the Dasharo Certification Lab, ensuring that the firmware binary
+meets the strict standards established by the program. By aligning with the
+Dasharo values, the certification program improves the overall posture of the
+open-source firmware ecosystem, making it long-term maintainable, sustainable,
+and trustworthy and providing specific service level agreements and warranties
+to commercial customers and the community.
+
+## What is DCP-approved firmware?
+
+The Dasharo-certified firmware provides long-term maintenance over ten years
+after the CPU microarchitecture release, which means that OEM, ODM, hardware
+vendors, and other companies can rely on the firmware for a long time without
+worrying about end-of-life issues. Moreover, DCP-approved firmware vendors must
+provide professional support channels to ensure that other business entities
+can rely on those channels for long-term support regarding firmware and
+software.
+
+The validation process for DCP firmware is transparent. Test results and bug
+reports are always publicly available, allowing the community to continually
+identify issues and improve the firmware. However, in case of a security
+embargo, the results can be kept under a strict but well-defined policy,
+ensuring the security of the firmware.
+
+DCP-approved firmware respects privacy by allowing the deactivation of
+potentially malicious components, like ME or PSP, to the level permitted by the
+current state of technology. This feature is essential, especially in
+increasing data privacy concerns. DCP-approved firmware also respects the
+liberty of hardware owners to repair and transfer ownership without risking any
+personally identifiable data.
+
+DCP-approved firmware should provide trustworthiness. It means that known good
+measurements of each boot phase are publicly available and stored in a
+tamper-resistant medium like TPM during the boot process. Based on publicly
+available reference measurements, firmware executed during the boot process can
```

Comment:
#future

Reply:
@macpijan please let me know if this works for you: https://github.com/Dasharo/docs/pull/507/commits/9f53835fe3d5e1dc2f18fd65021cb9b99bdabdf7